website/docs/d/cbr_zone.html.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cbr_zone"
+description: |-
+  Get information about cbr_zone
+subcategory: "Context Based Restrictions"
+---
+
+# ibm_cbr_zone
+
+Provides a read-only data source for cbr_zone. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_cbr_zone" "cbr_zone" {
+	zone_id = "zone_id"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `zone_id` - (Required, Forces new resource, String) The ID of a zone.
```

Comment:
Please remove `Forces new resource`, datasource ll not have any force new resource

Reply:
removed Forces new resource,